integration-tests/python-enclave/enclave/build.gradle

```diff
@@ -5,4 +5,15 @@ plugins {
 conclave {
     productID = 100
     revocationLevel = 2
+    maxThreads = 32
```

Comment:
Why this many? Is this a requirement from Python?

Reply:
It does not run otherwise and it will ask you to increase that. We have a similar number in the Python examples in Gramine.

CC @ac101m

Reply:
Can you add a comment to that effect here. This means the plugin will need to have different default values depending on whether its graalvm, gramine or python. I'll create a ticket for that.

Reply:
https://r3-cev.atlassian.net/browse/CON-1223